docs/HowTo/Deploy/Monitoring-Performance.md

```diff
@@ -18,6 +18,8 @@ To specify the host and port on which Prometheus accesses Besu, use the [`--metr
 [`--metrics-port`](../../Reference/CLI/CLI-Syntax.md#metrics-port) options. 
 The default host and port are 127.0.0.1 and 9545.
 
+Prometheus requires 3M of space per node per hour for metrics, with a `scrape_interval` of 15s.
```

Comment:
Could you specify the type of node (what consensus?) that you used and the block time (how often blocks are created changes the quantity of data I guess)?

Reply:
Is M Megabyte?  I'd use MB per the Microsoft Style Guide - https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/bits-bytes-terms

Reply:
Type of node and block period do not affect the amount of data.  How often Prometheus is configured to scrape does.

Reply:
I'd prefer to provide only the necessary information and omit the extras that might slow down users. @NicolasMassart Do you have additional thoughts about this issue?